sds/src/scanner/metrics.rs

```diff
@@ -0,0 +1,25 @@
+use crate::labels::{Labels, NO_LABEL};
+use metrics::{counter, Counter};
+
+pub struct Metrics {
+    pub false_positive_excluded_attributes: Counter,
+}
+
+const TYPE: &str = "type";
+
+impl Metrics {
+    pub fn new(labels: &Labels) -> Self {
+        Metrics {
+            false_positive_excluded_attributes: counter!(
+                "false_positive.excluded_attributes",
+                labels.clone_with_labels(Labels::new(&[(TYPE, "excluded_attributes".to_string())]))
+            ),
+        }
+    }
+}
+
+impl Default for Metrics {
+    fn default() -> Self {
+        Metrics::new(&NO_LABEL)
+    }
```

Comment:
I'm not sure it's needed, maybe we don't need a default here.

Reply:
Removed this in [Remove Default implementation + rename metric + remove extra labels](https://github.com/DataDog/dd-sensitive-data-scanner/pull/40/commits/382e492e80b97ab6c57f6672f5e028ab21e40ad9)